docs/source/Rules/HTTPResponseParser.rst

```diff
@@ -0,0 +1,239 @@
+.. _HTTPResponseParser:
+
+####################
+HTTP Response Parser
+####################
+
+Introduction
+============
+
+The HTTP Response Parser are a set of parsers that can be used to parse the response of an HTTP request. They extract specific information from the response and use it in rules by generating specific events.
+
+.. note:: These parsers are disabled by default and only available in max builds or custom builds where you have to enable them with either ``#define FEATURE_THINGSPEAK_EVENT 1``, ``#define FEATURE_OPENMETEO_EVENT 1`` or ``#define FEATURE_JSON_EVENT 1`` in the config.h file.
+
+The Thingspeak Parser
+---------------------
+
+* ``SendToHTTP`` generates an event with the response of a thingspeak request (https://de.mathworks.com/help/thingspeak/readlastfieldentry.html & https://de.mathworks.com/help/thingspeak/readdata.html)
```

Comment:
These urls seem to redirect to a language specific version, depending on the user location, even though the content is English (AFAICS).
Maybe it's more universal if `www` is used instead of `de`.

Reply:
true.. www seems to do the trick